addons/twelve-hour-time/addon.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "12 hour time",
+  "description": "Changes forum times into the 12 hour time equivalent",
+  "tags": ["community", "forums"],
+  "enabledByDefault": false,
+  "credits": [
+    {
+      "name": "god286/webdev03",
+      "link": "https://github.com/webdev03"
+    },
+    {
+      "name": "JaydenDev",
+      "link": "https://github.com/JaydenDev"
+    },
+    {
+      "name": "towerofnix",
+      "link": "https://github.com/towerofnix"
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "changer.js",
+      "matches": ["https://scratch.mit.edu/discuss/topic/*"]
```

Comment:
use word matcher

Reply:
Oh sorry, I didn't realise that was a thing.
(I changed the URL name.. LOL..)